### ISSUE TITLE
IL code is messed up on Attribute copy

### ISSUE THREAD

**Issue by reflection-emit** [Capgemini/Cauldron#19]
IL code is messed up if an attribute is copied to the actual code... If the attribute constructor has param arrays.

**Comment by reflection-emit** [comment]
Weaver created this
```
	IL_0020: nop
	IL_0021: ldc.i4.1
	IL_0022: newarr [mscorlib]System.Object
	IL_0027: stloc.s 6
	IL_0029: ldloc.s 6
	IL_002b: ldc.i4.0
	IL_002c: ldarg.1
	IL_002d: stelem.ref
	IL_002e: ldc.i4.4
	IL_002f: ldc.i4.0
	IL_0030: newarr [mscorlib]System.Type
	IL_0035: dup
	IL_0036: newobj instance void 
```
instead of this
```
	IL_0000: nop
	IL_0001: ldc.i4.4
	IL_0002: call !!0[] [mscorlib]System.Array::Empty<class [mscorlib]System.Type>()
	IL_0007: newobj instance void [
```

**Comment by reflection-emit** [comment]
fixed in 1.1.0
